src/zero/boot.clj

```diff
@@ -112,10 +112,10 @@
             (let [out (io/file resources (.getName in))]
               (io/make-parents out)
               (io/copy in out)))]
-    (let [config (clone "pipeline-config" "zero/environments.clj")]
+    (let [config (clone "pipeline-config" "wfl/environments.clj")]
       (stage (clone "dsde-pipelines" "tasks/CopyFilesFromCloudToCloud.wdl"))
       (util/shell-io! "git" "-C" (.getParent config)
-                      "checkout" "e44decd39db39725f79bddc42d944181d0621b1d")
+                      "checkout" "a51348d8f755a2c92aad84cc94a9235d5cc90450")
```

Comment:
Reminder to update this commit hash, now that the pipeline-config PR is merged :octocat:

Reply:
Thanks!!